environment-dev.yml

```diff
@@ -37,10 +37,10 @@ dependencies:
   - pandoc==2.1.3
   - pylint
   - flake8
+  - nbstripout
```

Comment:
I'm surprised that it's useful to have a dependency to strip output cells.
Shouldn't it be < 5 lines to do this via the `nbconvert` or `nbformat` API to loop over output cells and make them empty?
Of course, if it's nontrivial / useful to use `nbstripout`, OK to add the dependency.

Reply:
ok, done.
anyways, I have foreseen for developers to use `nbstripout` to configure a git filter before committing notebooks, so thought would be installed in the devel environment. But I agree, better to reduce dependencies.